src/cli/db-migration.ts

```diff
@@ -0,0 +1,154 @@
+import fs from 'fs'
+import path from 'path'
+import { flags } from '@oclif/command'
+import { OutputFlags } from '@oclif/parser'
+
+import DbMigration from '../../migrations'
+import { BaseCLICommand } from '../utils'
+import { sequelizeFactory } from '../sequelize'
+import config from 'config'
+
+const MigrationTemplate = `import { QueryInterface } from 'sequelize'
+import { Sequelize } from 'sequelize-typescript'
+
+export default {
+  // eslint-disable-next-line require-await
+  async up (queryInterface: QueryInterface, sequelize: Sequelize): Promise<void> {
+    return Promise.reject(Error('Not implemented'))
+  },
+  // eslint-disable-next-line require-await
+  async down (queryInterface: QueryInterface, sequelize: Sequelize): Promise<void> {
+    return Promise.reject(Error('Not implemented'))
+  }
+}
+`
+
+export default class DbMigrationCommand extends BaseCLICommand {
+  static hidden: boolean;
+  static flags = {
+    ...BaseCLICommand.flags,
+    db: flags.string({ description: 'database connection URI', env: 'RIFM_DB' }),
+    up: flags.boolean({
+      char: 'u',
+      description: 'Migrate DB',
+      exclusive: ['down', 'generate']
+    }),
+    down: flags.boolean({
+      char: 'd',
+      description: 'Undo db migrations',
+      exclusive: ['up', 'generate']
+    }),
+    generate: flags.string({
+      char: 'd',
+      description: 'Generate migrations',
```

Comment:
This description is not helpful at all. It requires some string, but I don't know what I should put there? I thought that it is just a boolean flag originally.

Reply:
I also thought that this generate actual migration and not just a template. Maybe this info should be given as well.

Reply:
Done